static/app/components/discover/transactionsTable.tsx

```diff
@@ -92,6 +93,25 @@ class TransactionsTable extends PureComponent<Props> {
         );
       }
 
+      if (column.key === 'replayId') {
+        return (
+          <HeadCellContainer key={index}>
+            <SortLink
+              align={align}
+              title={
+                <StyledGridCell>
+                  <StyledIconPlay size="xs" />
```

Comment:
Can we move this icon so it's inside the table, not in the header?

The header will have icons for sorting, it'll be busy up there

Reply:
Screenshots updated!